submissions/factorio_draftsman/main.md

```diff
@@ -0,0 +1,200 @@
+# Introduction
+
+A couple months ago, I decided that I wanted to try my hand at making a **Self-expanding factory** in Factorio. After seeing an number of [impressive](https://www.youtube.com/watch?v=xF--1XdcOeM) [examples](https://www.youtube.com/watch?v=KNSBerFe_H8), I was inspired to have a go at the problem. I already had an outline of the logic, and how the factory would keep track of itself, as well as lofty ideas on impressive things that I could make it do. The only trouble was that I had actually never used combinators before, and I was planning on using them for the actual decision-making.
+
+That's no problem though, we just create a scrap world, switch to the editor and start playing around!
+
+![death](./img/map_view_of_madness.png)
+
+![additional death](./img/madness.png "Any attempt to understand the meaningless scribbles of a madman will only result in disappointment and despair.")
+
+![too much playtime](./img/play_time.png "ROOKIE NUMBERS")
```

Comment:
I think we can turn this into a ["gallery"](https://alt-f4.blog/ALTF4-X/#gallery) so that the images don't take to much space proportially. All you have to do is add a little comment and we can add it later.
```suggestion
// Conor, these three images should be a gallery
![death](./img/map_view_of_madness.png)

![additional death](./img/madness.png "Any attempt to understand the meaningless scribbles of a madman will only result in disappointment and despair.")

![too much playtime](./img/play_time.png "ROOKIE NUMBERS")
```